src/Integration.Vsix/Telemetry/GeneratedCode/SqmOleCommandTarget.cs

```diff
@@ -33,6 +33,7 @@ internal enum SonarLintSqmCommandIds
         ToggleShowAllProjectsCommandCommandId = 0x306,
         DontWarnAgainCommandCommandId = 0x307,
         FixConflictsCommandCommandId = 0x308,
+        UpdateBindingCommandFromErrorListCommandId = 0x400,
```

Comment:
We should have an agreed upon range for these command IDs. In the work I'm doing I'm using the 0x400 range.
I think telemetry should go up to the 'higher' end, to make room for real commands to expand, i.e., 0x99###, so you can more easily tell it's a telemetry command from the first two digits.